bindings/node/lib/errors.js

```diff
@@ -49,8 +49,27 @@ class MongoCryptCreateEncryptedCollectionError extends MongoCryptError {
   }
 }
 
+/**
+ * @class
+ * An error indicating that mongodb-client-encryption failed to auto-refresh Azure KMS credentials.
+ */
+class MongoCryptAzureKMSRequestError extends Error {
```

Comment:
Intentionally not extending from `MongoCryptError`?

Reply:
No, good catch 👍